base_business_document_import/models/business_document_import.py

```diff
@@ -222,97 +261,116 @@ def _hook_match_partner(
         return False
 
     @api.model
-    def _match_shipping_partner(self, shipping_dict, partner, chatter_msg):
+    def _match_shipping_partner(
+        self, partner_dict, partner, chatter_msg, domain=None, raise_exception=True
```

Comment:
```suggestion
        self, partner_dict, partner, chatter_msg, domain=None
```
Seems not to be used anywhere. And can probably be handled by a `try...except` in function using `super` (if this was added in anticipation)